server/routes.go

```diff
@@ -721,12 +721,61 @@ func GetModelInfo(req api.ShowRequest) (*api.ShowResponse, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	resp.Modelfile = mf
 
+	ggmlData, err := getGGMLData(model)
+	if err != nil {
+		return nil, err
+	}
+	resp.ModelInfo = string(ggmlData)
+
 	return resp, nil
 }
 
+func getGGMLData(model *Model) ([]byte, error) {
+	f, err := os.Open(model.ModelPath)
+	if err != nil {
+		return nil, err
+	}
+
+	ggml, _, err := llm.DecodeGGML(f)
+	if err != nil {
+		return nil, err
+	}
+
+	kv := ggml.KV()
+	var keys []string
+	for k := range kv {
+		keys = append(keys, k)
+	}
+
+	kvMap := make(map[string]any)
+
+	for _, k := range keys {
+		val := kv[k]
+
+		switch v := val.(type) {
+		case []interface{}:
+			if len(v) > 5 {
+				kvMap[k] = []string{}
```

Comment:
It's confusing to see an empty JSON list as the value. This should have something that indicates the type and size of the value, something like `(list: 280K items)`